src/Mvc/Mvc.Abstractions/src/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,3 @@
 #nullable enable
+Microsoft.AspNetCore.Mvc.ModelBinding.BindingInfo.ServiceKey.get -> object?
+Microsoft.AspNetCore.Mvc.ModelBinding.BindingInfo.ServiceKey.set -> void
```

Comment:
@benjaminpetit here's the modification to the API file that the build is complaining about. Normally we don't change these files in `release` branches, but since this is the RC1 branch it's fine. Ping me or a manager when this is approved & CI failures are down to just this, and we can force-merge it for you (note I'll be OOF for a week after about 3:30 PST today)